adapter/pkg/config/log_types.go

```diff
@@ -28,6 +28,10 @@ type accessLog struct {
 	Format  string
 }
 
+type routerDebugLog struct {
```

Comment:
Shall we rename it to debugLogs? Now we have accessLogs. Also add the config to the log_config.toml